Changelog.md

```diff
@@ -4,6 +4,7 @@ __New Features__
 - Allows heating/cooling seasons that don't span the entire year.
 - Allows generating timeseries unmet hours for heating and cooling.
 - Allows CSV timeseries output to be formatted for use with the DView application.
+- Allows modeling room/window heat pump and room air conditioner with heating.
```

Comment:
Need to let users know that there's a breaking change related to PTAC w/ heating.